packages/vscode/src/plugins/prisma-postgres-manager/ConnectionStringStorage.ts

```diff
@@ -0,0 +1,120 @@
+import type { SecretStorage } from 'vscode'
+import z from 'zod'
+
+// Storing connection strings in this JSON structure under a single key in the secret storage.
+// This allows for easy deletion of multiple entries when a project is deleted or the user logs out from a workspace.
+const StoredConnectionStringsSchema = z.object({
+  workspaces: z.record(
+    z.string(), // workspaceId
```

Comment:
Might be more descriptive if you make these schemas named constants instead of writing these comments:
```ts
const WorkspaceIdSchema = z.string()
const ProjectIdSchema = z.string()
const DatabaseIdSchema = z.string()
```

Reply:
good idea - will follow up with :+1: